integration/tests/epochs/suite.go

```diff
@@ -543,6 +541,24 @@ func (s *Suite) StakeNewNode(ctx context.Context, env templates.Environment, rol
 	return info, testContainer
 }
 
+// getContainerToReplace return a container from the network, make sure the container is not a ghost
+func (s *Suite) getContainerToReplace(role flow.Role) *testnet.Container {
+	nodes := s.net.ContainersByRole(role)
+	require.True(s.T(), len(nodes) > 0)
+
+	if role != flow.RoleAccess {
+		return nodes[0]
+	}
+
```

Comment:
I think it's actually possible to create a ghost node that isn't an access node, even if we don't currently do it, so we could just remove this conditional.
```suggestion
```

Reply:
https://github.com/onflow/flow-go/pull/1728/commits/55810e1bfedd693005ff41ef5322af68b0a6d5dd